mongodbatlas/resource_mongodbatlas_project_api_key_test.go

```diff
@@ -268,3 +304,37 @@ func testAccMongoDBAtlasProjectAPIKeyConfigMultiple(orgID, projectName, descript
 
 	`, orgID, projectName, description, roleNames)
 }
+
+func testAccMongoDBAtlasProjectAPIKeyConfigOrgRoles(orgID, firstProjectName, secondProjectName, description string) string {
+	return fmt.Sprintf(`
+		resource "mongodbatlas_project" "test" {
+			name   = %[2]q
+			org_id = %[1]q
+		}
+
+		resource "mongodbatlas_project" "testProject" {
+			name   = %[3]q
+			org_id = %[1]q
+		}
+
+		resource "mongodbatlas_project_api_key" "test" {
+			project_id     = mongodbatlas_project.test.id
+			description  = %[4]q
+			project_assignment  {
```

Comment:
Can we please update/add tests with multiple `project_assignment`s since it is a TypeSet? None of the tests for this resource seem to do be doing that to correctly test updates/reads etc.

Reply:
updated